### PR TITLE
bump

### PR DESCRIPTION
**Pull request by jordens**
- build.rs is default
- bump dependencies

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1011,9 +947,9 @@ checksum = "388a1df253eca08550bef6c72392cfe7c30914bf41df5269b68cbd6ff8f570a3"
 
 [[package]]
 name = "sequential-storage"
-version = "0.6.1"
+version = "0.6.2"
```
Comment (ryan-summers) [inline Cargo.lock:950]: I think this bump is breaking the build

Comment (ryan-summers) [reply Cargo.lock:950]:
```
error[E0446]: private type `PageState` in public interface
   --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/sequential-storage-0.6.2/src/item.rs:405:1
    |
405 | / pub fn is_page_empty<S: NorFlash>(
406 | |     flash: &mut S,
407 | |     flash_range: Range<u32>,
408 | |     page_index: usize,
409 | |     page_state: Option<PageState>,
410 | | ) -> Result<bool, Error<S::Error>> {
    | |__________________________________^ can't leak private type
    |
   ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/sequential-storage-0.6.2/src/lib.rs:204:1
    |
204 |   enum PageState {
    |   -------------- `PageState` declared as private
    ```

Not sure if this is MSRV related or not?

Comment (jordens) [reply Cargo.lock:950]: yeah. let's take 0.6.2 as there appear to be some potentially important bug fixes in there. I really don't get why newer compilers don't complain about that type leakage anymore.
I wonder if we should just say "latest stable at release time" as the MSRV policy.

**Comment by jordens** [comment]
needs github branch protection updates/override on merge

**Review by ryan-summers — APPROVED** [review]

**Pull request merged**